test/monitoring/monitoring.go

```diff
@@ -59,10 +60,10 @@ func Cleanup(pid int) error {
 }
 
 // PortForward sets up local port forward to the pod specified by the "app" label in the given namespace
-func PortForward(logf logging.FormatLogger, podList *v1.PodList, localPort, remotePort int, namespace string) (int, error) {
+func PortForward(logf logging.FormatLogger, podList *v1.PodList, localPort, remotePort int, namespace string, stdout io.Writer, stderr io.Writer) (int, error) {
```

Comment:
Maybe add a comment that stdout/stderr are where the results of the command are written. As in, why it's different from logf.

Reply:
Actually, can't we use the getcombinedoutput?
https://golang.org/src/os/exec/exec.go?s=15990:16036#L547

Reply:
is it because of the background bits?

Reply:
Since the kubectl should be fairly fast, I wonder if it is necessary to run it in the background instead of waiting for it to return and fail fast instead of later?

Reply:
It has to run in background because the process remains open

Reply:
yeah, I realized that, forgot to resolve the conversation, but stamped it :)